static-build/style.css

```
@@ -0,0 +1,481 @@
+/* ===== Fonts ===== */
+@font-face {
+  font-family: 'Lekton';
```

Stylelint (17.9.0)

[error] Expected no quotes around "Lekton" (font-family-name-quotes)

(font-family-name-quotes)